crates/rust-analyzer/src/config.rs

```diff
@@ -762,7 +762,11 @@ config_data! {
         /// though Cargo might be the eventual consumer.
         vfs_extraIncludes: Vec<String> = vec![],
 
-        /// Exclude imports from symbol search.
+        /// Exclude all imports from workspace symbol search.
```

Comment:
It may be worth defining what symbol search is (eg specifically where this ends up in the ui)

Reply:
This doesn't seem like the right location to explain symbol search to me. In my opinion that should be the job of the editor that exposes the feature.